.github/workflows/merge-main.yml

```diff
@@ -105,14 +105,15 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Deploys
-        uses: bcgov-nr/action-deployer-openshift@v0.1.0
+        uses: bcgov-nr/action-deployer-openshift@v0.2.0-rc1
```

Comment:
Please give me a little longer to finish off a non-rc release.  Otherwise, great!  :)